spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -710,6 +713,9 @@ Instance<Object> createInstance();
 
 Instances of dependent scoped beans obtained with this `Instance` object must be explicitly released by calling `Instance.destroy()` method.
 
+When performing lookup of bean instance with the returned `Instance`, the qualifiers specified in the `Instance.select()` method are used to perform typesafe resolution.
```

Comment:
I would leave out this sentence - it's obvious, and add only:
If no qualifier is passed to the `Instance.select()` method the `@Default` qualifier is assumed.